ironfish-cli/src/commands/wallet/import.ts

```diff
@@ -48,9 +48,9 @@ export class ImportCommand extends IronfishCommand {
 
     const client = await this.sdk.connectRpc()
 
```

Comment:
We will be able to get rid of a lot (if not all) of this code once we update the RPC and delete this code from the CLI.